packages/tools/devtools/devtools-browser-extension/package.json

```
@@ -29,16 +29,17 @@
 		"prettier": "prettier --check . --ignore-path ../../../../.prettierignore",
 		"prettier:fix": "prettier --write . --ignore-path ../../../../.prettierignore",
 		"rebuild": "npm run clean && npm run build",
-		"start:client": "webpack serve --config webpack.test.js",
-		"start:tinylicious": "tinylicious",
+		"start:client:test": "webpack serve --config webpack.test.js",
 		"test": "npm run test:mocha",
```

We should probably update this to run both `test:mocha` and `test:jest`.

```suggestion
		"test": "concurrently npm:test:mocha npm:test:jest",
```

---

I would verify that the above syntax is correct, and that it actually works okay. But I think it's right :)

---

I'm not sure which script will run `npm run test` but client-package pipeline will run `npm run jest:test` for all directories by `npm run ci:jest:test`. Let me verify the mocha tests get calls in some scripts. I think if they both get calls and are called separately then it's okay to keep it that way.

---

The CI pipelines don't use the `test` script, they strictly use `test:mocha`, `test:jest`, etc. But if we run `npm run test` in this directory, it would be nice to have it run all of the tests.

---

I see. Updated it!